undefined: prometheus.Handler

How can I download or create the binary?
When executing:

`go get -u https://github.com/ferringb/ldap_exporter`

Return: 

github.com/ferringb/ldap_exporter (download)
github.com/Masterminds/sprig (download)
github.com/Masterminds/goutils (download)
github.com/Masterminds/semver (download)
github.com/google/uuid (download)
github.com/huandu/xstrings (download)
github.com/imdario/mergo (download)
github.com/mitchellh/copystructure (download)
github.com/mitchellh/reflectwalk (download)
github.com/spf13/cast (download)
Fetching https://golang.org/x/crypto/scrypt?go-get=1
Parsing meta tags from https://golang.org/x/crypto/scrypt?go-get=1 (status code 200)
get "golang.org/x/crypto/scrypt": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/scrypt?go-get=1
get "golang.org/x/crypto/scrypt": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto?go-get=1
Parsing meta tags from https://golang.org/x/crypto?go-get=1 (status code 200)
golang.org/x/crypto (download)
Fetching https://golang.org/x/crypto/pbkdf2?go-get=1
Parsing meta tags from https://golang.org/x/crypto/pbkdf2?go-get=1 (status code 200)
get "golang.org/x/crypto/pbkdf2": found meta tag get.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/pbkdf2?go-get=1
get "golang.org/x/crypto/pbkdf2": verifying non-authoritative meta tag
github.com/prometheus/client_golang (download)
github.com/beorn7/perks (download)
github.com/golang/protobuf (download)
github.com/prometheus/client_model (download)
github.com/prometheus/common (download)
github.com/matttproud/golang_protobuf_extensions (download)
github.com/prometheus/procfs (download)
github.com/sirupsen/logrus (download)
Fetching https://golang.org/x/sys/unix?go-get=1
Parsing meta tags from https://golang.org/x/sys/unix?go-get=1 (status code 200)
get "golang.org/x/sys/unix": found meta tag get.metaImport{Prefix:"golang.org/x/sys", VCS:"git", RepoRoot:"https://go.googlesource.com/sys"} at https://golang.org/x/sys/unix?go-get=1
get "golang.org/x/sys/unix": verifying non-authoritative meta tag
Fetching https://golang.org/x/sys?go-get=1
Parsing meta tags from https://golang.org/x/sys?go-get=1 (status code 200)
golang.org/x/sys (download)
Fetching https://gopkg.in/alecthomas/kingpin.v2?go-get=1
Parsing meta tags from https://gopkg.in/alecthomas/kingpin.v2?go-get=1 (status code 200)
get "gopkg.in/alecthomas/kingpin.v2": found meta tag get.metaImport{Prefix:"gopkg.in/alecthomas/kingpin.v2", VCS:"git", RepoRoot:"https://gopkg.in/alecthomas/kingpin.v2"} at https://gopkg.in/alecthomas/kingpin.v2?go-get=1
gopkg.in/alecthomas/kingpin.v2 (download)
github.com/alecthomas/template (download)
github.com/alecthomas/units (download)
Fetching https://gopkg.in/ldap.v2?go-get=1
Parsing meta tags from https://gopkg.in/ldap.v2?go-get=1 (status code 200)
get "gopkg.in/ldap.v2": found meta tag get.metaImport{Prefix:"gopkg.in/ldap.v2", VCS:"git", RepoRoot:"https://gopkg.in/ldap.v2"} at https://gopkg.in/ldap.v2?go-get=1
gopkg.in/ldap.v2 (download)
Fetching https://gopkg.in/asn1-ber.v1?go-get=1
Parsing meta tags from https://gopkg.in/asn1-ber.v1?go-get=1 (status code 200)
get "gopkg.in/asn1-ber.v1": found meta tag get.metaImport{Prefix:"gopkg.in/asn1-ber.v1", VCS:"git", RepoRoot:"https://gopkg.in/asn1-ber.v1"} at https://gopkg.in/asn1-ber.v1?go-get=1
gopkg.in/asn1-ber.v1 (download)
Fetching https://gopkg.in/yaml.v2?go-get=1
Parsing meta tags from https://gopkg.in/yaml.v2?go-get=1 (status code 200)
get "gopkg.in/yaml.v2": found meta tag get.metaImport{Prefix:"gopkg.in/yaml.v2", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v2"} at https://gopkg.in/yaml.v2?go-get=1
gopkg.in/yaml.v2 (download)
github.com/marcusteixeira/ldap_exporter
# github.com/marcusteixeira/ldap_exporter
**go/src/github.com/marcusteixeira/ldap_exporter/main.go:161:28: undefined: prometheus.Handler**

--- a/main.go
+++ b/main.go
@@ -13,6 +13,7 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"gopkg.in/ldap.v2"
 )
 
@@ -158,6 +159,6 @@ func main() {
 	prometheus.MustRegister(e)
 
 	log.Infof("starting server; telemetry accessible at %s%s", *listen, *metricsPath)
-	http.Handle(*metricsPath, prometheus.Handler())
+	http.Handle(*metricsPath, promhttp.Handler())
 	log.Fatal(http.ListenAndServe(*listen, nil))
 }

